tests/test_main.py

```diff
@@ -956,6 +956,44 @@ class Bar(BaseModel):
     assert dict(m) == {'c': 3, 'd': Foo()}
 
 
+def test_model_export_nested_list():
+    class Foo(BaseModel):
+        a: int = 1
+        b: int = 2
+
+    class Bar(BaseModel):
+        c: int
+        foos: List[Foo]
+
+    m = Bar(c=3, foos=[Foo(), Foo()])
+    exclusion = {idx: {'a'} for idx in range(len(m.foos))}
```

Comment:
better to be explicit here (we're in a test)
```suggestion
    exclusion = {0: {'a'}, 1: {'a'}}
```